link-hint.el

```diff
@@ -73,7 +73,8 @@
     ;; generic
     link-hint-button
     link-hint-text-url
-    link-hint-file-link)
+    link-hint-file-link
+    link-hint-treemacs)
```

Comment:
The generic types of links go at the bottom. Even though you've disabled the generic button type in treemacs mode, could you putton the new type before the "generic" comment (e.g. right after the woman button type) for consistency?